packages/ddp-client/common/livedata_connection.js

```diff
@@ -1708,6 +1725,19 @@ export class Connection {
       return true;
     });
 
+    if (!self.shouldRetryMethods) {
+        self.shouldRetryMethods = true;
+        // Run the callback on all remaining method calls with the error as the parameter
+        if (isEmpty(oldOutstandingMethodBlocks)) return;
+
+        while (oldOutstandingMethodBlocks.length) {
+            const methods = oldOutstandingMethodBlocks.pop().methods;
+            if (!isEmpty(methods)) {
```

Comment:
No need for this extra check either, `forEach` would simply do nothing if it's empty.

Reply:
fixed